### ISSUE TITLE
Fix some UX/UI issues on the texts translation UI

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#146]
## UI issues
There are some part of the design of the texts translation UI that doesn't respect the Figma's mockups (see the caption below):
- [ ] In the bottom component that make possible to switch to the target language, the text part of each line should be medium and not regular.
- [x] The same text should get the color text100 when the user is hovering the line.

![Image](https://github.com/PokemonWorkshop/PokemonStudio/assets/20935418/a4367b6f-6ef9-49b7-a8fc-360e37193cd8)

## UX issues
- [x] We should be able to use LEFT/RIGHT arrows or CTRL + LEFT / CTRL + RIGHT arrows to go to the previous or next text entry. This to improve the accessibility too.
